### PR TITLE
[zh] Sync Tutorial update-intro.

### PR DESCRIPTION
**Pull request by 1000Delta**
Sync with:

```
/docs/tutorials/kubernetes-basics/update/update-intro.html
```

### CODE REVIEW

**Comment by 1000Delta** [comment]
See [Preview](https://deploy-preview-43956--kubernetes-io-main-staging.netlify.app/zh-cn/docs/tutorials/kubernetes-basics/update/update-intro/).

File: content/zh-cn/docs/tutorials/kubernetes-basics/update/update-intro.html [hunk 1]
```diff
@@ -41,49 +42,52 @@ <h3>目标</h3>
...
+
             <!--
             <p>Users expect applications to be available all the time and developers are expected to deploy new versions of them several times a day. In Kubernetes this is done with rolling updates. <b>Rolling updates</b> allow Deployments' update to take place with zero downtime by incrementally updating Pods instances with new ones. The new Pods will be scheduled on Nodes with available resources.</p>
             -->
-            <p>用户希望应用程序始终可用，而开发人员则需要每天多次部署它们的新版本。在 Kubernetes 中，这些是通过滚动更新（Rolling Updates）完成的。 <b>滚动更新</b> 允许通过使用新的实例逐步更新 Pod 实例，零停机进行 Deployment 更新。新的 Pod 将在具有可用资源的节点上进行调度。</p>
-            
+            <p>用户希望应用程序始终可用，而开发人员则需要每天多次部署它们的新版本。
+                在 Kubernetes 中，这些是通过滚动更新（Rolling Updates）完成的。 
+                <b>滚动更新</b> 允许通过使用新的实例逐步更新 Pod 实例，零停机进行 Deployment 更新。
+                新的 Pod 将在具有可用资源的节点上进行调度。</p>
```
Comment (tengqm) [inline content/zh-cn/docs/tutorials/kubernetes-basics/update/update-intro.html:52]:
```suggestion
                新的 Pod 将被调度到具有可用资源的节点上。</p>
```

File: content/zh-cn/docs/tutorials/kubernetes-basics/update/update-intro.html [hunk 2]
```diff
@@ -173,16 +180,145 @@ <h2 style="color: #3771e3;">滚动更新概述</h2>
...
+               <!--
+               <p>Create an environment variable called <tt>NODE_PORT</tt> that has the value of the Node port assigned:</p>
+               -->
+               <P>创建名为 <tt>NODE_PORT</tt> 的环境变量，值为已被分配的 Node 端口：</P>
+               <p><code><b>export NODE_PORT="$(kubectl get services/kubernetes-bootcamp -o go-template='{{(index .spec.ports 0).nodePort}}')"</b></code><br />
+                  <code><b>echo "NODE_PORT=$NODE_PORT"</b></code></p>
+               <!--
+               <p>Next, do a <code>curl</code> to the exposed IP and port:</p>
+               -->
+               <p>接下来，对暴露的 IP 和端口执行 <code>curl</code>：</p>
```
Comment (tengqm) [inline content/zh-cn/docs/tutorials/kubernetes-basics/update/update-intro.html:235]:
```suggestion
               <p>接下来，针对所暴露的 IP 和端口执行 <code>curl</code>：</p>
```

**Comment by windsonsea** [comment]
Thanks.
/approve

**Pull request merged**